### ISSUE TITLE
How to increase X-axis range

### ISSUE THREAD

**Issue by tanuj83** [tulios/burrow-stats#7]
Is there any way to extend time of X-Axis to keep 2 days data at least?

**Comment by tulios** [comment]
`cacheBinSize` is the size of time frames it will hold, take a look at the readme for more details.

```
{
  "burrowStatsOptions": {
    "pollInterval": 60,
    "cacheBinSize": 30,
    "envLabel": "Staging",
    "envLabelColor": "white",
    "envLabelBgColor": "rgba(0, 137, 207, 1)"
  }
}
```

**Comment by tanuj83** [comment]
Thank you Tulios, I could see that before, but seems it saves cache in browser, as when I access page from other browser it starts fresh graph, no history

**Comment by tulios** [comment]
@tanuj83 you are correct, this project was created to serve as an easy to setup dashboard. It saves the data locally.
